server/repository/src/migrations/v1_06_00/indexes.rs

```diff
@@ -46,7 +46,6 @@ pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
         CREATE INDEX index_invoice_clinician_id ON invoice (clinician_id);
         CREATE INDEX index_plugin_data_store_id ON plugin_data (store_id);
         CREATE INDEX index_program_enrolment_program_id ON program_enrolment (program_id);
-        CREATE INDEX index_clinician_store_join_clinician_id ON clinician_store_join (clinician_id);
```

Comment:
Was causing errors even when I tried to drop index... so have removed since it's in the same migration version anyway (for now)